src/actions/assetsActions.js

```diff
@@ -125,11 +124,11 @@ export const sendAssetAction = (
     const accountId = getAccountId(activeAccount);
     const accountAddress = getAccountAddress(activeAccount);
 
-    const accountCollectibles = collectibles[accountId] || [];
+    const accountCollectiblesOnChain = accountCollectiblesSelector(getState())?.[chain] ?? [];
```

Comment:
```suggestion
    const accountCollectiblesOnChain = accountCollectiblesSelector(getState())?.[chain] ?? [];
```

Just curious here - are we evaluating the output of getState() as a whole to be truthy or falsey here?

Reply:
In case the output of `accountCollectiblesSelector` is falsey then it falls back to array

Reply:
Got it - cheers!